### PR TITLE
Fix session id auth: the data attribute in the HTML needs to match

### PR DESCRIPTION
**Pull request by diox**
Fixes https://github.com/mozilla/addons-code-manager/issues/2156

### CODE REVIEW

File: src/reducers/api.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ type SetAuthSessionIdPayload = {
 };
 
 export const actions = {
-  setAuthSessionId: createAction('SET_AUTH_TOKEN', (resolve) => {
+  setAuthSessionId: createAction('SET_AUTH_SESSION_ID', (resolve) => {
```
Comment (eviljeff) [inline src/reducers/api.tsx:12]: I found this with a search but tbh I'm not 100% what is it for - we don't reference it elsewhere in the code.

File: .env [hunk 2]
```diff
@@ -5,7 +5,7 @@ REACT_APP_REVIEWERS_HOST=https://reviewers.addons.mozilla.org
 
 # This is the placeholder used in the `index.html` file to inject the
 # authentication token on runtime.
-REACT_APP_AUTH_TOKEN_PLACEHOLDER=__AUTH_TOKEN__
+REACT_APP_USER_AUTH_SESSION_ID_PLACEHOLDER=__AUTH_TOKEN__
```
Comment (eviljeff) [inline .env:8]: Should we rename `__AUTH_TOKEN__` too?  It's not referenced anywhere else so 🤷

File: src/reducers/api.tsx (hunk 1 above)
Comment (eviljeff) [reply src/reducers/api.tsx:12]: I assume we're okay to change it then

Comment (bobsilverberg) [reply src/reducers/api.tsx:12]: It's just an internal constant to identify the action. It makes sense that it isn't used anywhere else.

**Review by bobsilverberg — APPROVED** [review]
Looks good, thanks @diox and @eviljeff for taking care of this!

**Pull request merged**